Riot/Managers/Theme/Themes/DarkTheme.swift

```diff
@@ -117,13 +117,12 @@ class DarkTheme: NSObject, Theme {
         applyStyle(onNavigationBar: navigationBar, withModernScrollEdgesAppearance: false)
     }
     
-    // Note: We are not using UINavigationBarAppearance on iOS 13/14 because of UINavigationBar directly including UISearchBar on their titleView that cause crop issues with UINavigationController pop.
     func applyStyle(onNavigationBar navigationBar: UINavigationBar,
                     withModernScrollEdgesAppearance modernScrollEdgesAppearance: Bool) {
         navigationBar.tintColor = tintColor
         
-        // On iOS 15 use UINavigationBarAppearance to fix visual issues with the scrollEdgeAppearance style.
-        if #available(iOS 15.0, *) {
+        // On iOS 13+ use UINavigationBarAppearance to fix visual issues with the scrollEdgeAppearance style.
```

Comment:
Technically this comment is still only referring to iOS 15, as 13 & 14 don't have a `scrollEdgeAppearance` property.